frontend/src/components/SearchPage/CourseItem.js

```diff
@@ -226,29 +232,29 @@ const CourseItem = ({ name, code, professors, metrics, UID }) => {
               </tr>
             </thead>
             <tbody>
-              {newProfessors.slice(0, 4).map(prof =>
+              {newProfessors.slice(0, 4).map(prof => (
```

Comment:
This is an issue with my code, but could you define a constant `COURSE_ITEM_LIST_LENGTH` or something at the top of this file so that it's easy to change the length of the list of professors? (Eventually I want to make it responsive but just for now)